src/main/java/io/ebean/dbmigration/ddlgeneration/platform/BaseTableDdl.java

```diff
@@ -741,6 +755,15 @@ public void generate(DdlWrite writer, AddColumn addColumn) throws IOException {
   public void generate(DdlWrite writer, DropTable dropTable) throws IOException {
 
     dropTable(writer.apply(), dropTable.getName());
+
+    if (hasValue(dropTable.getSequenceCol())
+        && platformDdl.getPlatform().getDbIdentity().isSupportsSequence()) {
+      String sequenceName = dropTable.getSequenceName();
+      if (!hasValue(sequenceName)) {
+        sequenceName = namingConvention.getSequenceName(dropTable.getName(), dropTable.getSequenceCol());
+      }
+      dropSequence(writer.apply(), sequenceName);
```

Comment:
When dropping a table, we should also drop the sequence. For this, I had to extend the DropTable XML-model, so that this will work only for further migrations.